### PR TITLE
Feature/works about me

### PR DESCRIPTION
**Pull request by Albinpirela**
##Add the works-about-me branch 
#Add style to branch
Captura de pantalla (35)
Captura de pantalla (37)
Captura de pantalla (38)

### CODE REVIEW

**Review by GerardoGomezG — APPROVED** [review]
Well done bro!
You're ready to merge this branch into "develop"
thank you @GerardoGomezG , I really appreciate your review, but I need you to please make one last review because I made some changes after the approval

File: index.html [hunk 1]
```diff
@@ -75,40 +75,40 @@ <h2>Skills</h2>
...
..."none" text-anchor="none" style="mix-blend-mode: normal"><path d="M0,256v-256h256v256z" id="bgRectangle"></path></g><g fill="#ffffff" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><g transform="scale(5.12,5.12)"><path d="M45.27344,2.32422c-0.1875,-0.20703 -0.45703,-0.32422 -0.73828,-0.32422h-39.07031c-0.28125,0 -0.55078,0.11719 -0.73828,0.32422c-0.19141,0.20703 -0.28516,0.48438 -0.25781,0.76563l3.51953,39.42578c0.03516,0.41406 0.32422,0.75781 0.72266,0.875l16.01172,4.57031c0.08594,0.02734 0.17969,0.03906 0.27344,0.03906c0.09375,0 0.18359,-0.01172 0.27344,-0.03906l16.02344,-4.57031c0.39844,-0.11719 0.68359,-0.46094 0.72266,-0.875l3.51563,-39.42578c0.02734,-0.28125 -0.06641,-0.55859 -0.25781,-0.76562zM36.84766,15.91797h-18.8125l0.44922,5.08984h17.91016l-1.34375,15.04297l-10.05859,3.03906l-0.09766,-0.03125l-9.94141,-3.01172l-0.54297,-6.12891h4.87109l0.21094,2.37891l5.55859,1.16406l5.45703,-1.16406l0.58203,-6.4375h-17.04297l-1.32422,-14.80469h24.55859z"></path></g></g>
```
Comment (GerardoGomezG) [inline index.html]: Hey dude!
Can you explain a little what these lines of codes where for?

File: index.html [hunk 2]
```diff
@@ -75,40 +75,40 @@ <h2>Skills</h2>
...
..."none" text-anchor="none" style="mix-blend-mode: normal"><path d="M0,256v-256h256v256z" id="bgRectangle"></path></g><g fill="#ff8303" fill-rule="nonzero" stroke="none" stroke-width="1" stroke-linecap="butt" stroke-linejoin="miter" stroke-miterlimit="10" stroke-dasharray="" stroke-dashoffset="0" font-family="none" font-weight="none" font-size="none" text-anchor="none" style="mix-blend-mode: normal"><g transform="scale(5.12,5.12)"><path d="M45.27344,2.32422c-0.1875,-0.20703 -0.45703,-0.32422 -0.73828,-0.32422h-39.07031c-0.28125,0 -0.55078,0.11719 -0.73828,0.32422c-0.19141,0.20703 -0.28516,0.48438 -0.25781,0.76563l3.51953,39.42578c0.03516,0.41406 0.32422,0.75781 0.72266,0.875l16.01172,4.57031c0.08594,0.02734 0.17969,0.03906 0.27344,0.03906c0.09375,0 0.18359,-0.01172 0.27344,-0.03906l16.02344,-4.57031c0.39844,-0.11719 0.68359,-0.46094 0.72266,-0.875l3.51563,-39.42578c0.02734,-0.28125 -0.06641,-0.55859 -0.25781,-0.76562zM36.84766,15.91797h-18.8125l0.44922,5.08984h17.91016l-1.34375,15.04297l-10.05859,3.03906l-0.09766,-0.03125l-9.94141,-3.01172l-0.54297,-6.12891h4.87109l0.21094,2.37891l5.55859,1.16406l5.45703,-1.16406l0.58203,-6.4375h-17.04297l-1.32422,-14.80469h24.55859z"></path></g></g>
```
Comment (GerardoGomezG) [inline index.html]: What's the difference with previous deleted lines of code?

File: index.html (hunk 1 above)
Comment (GerardoGomezG) [reply index.html]: Ok @Albinpirela
I've seen it's related to SVG images, so keep on moving man...
You're doing well!

File: index.html (hunk 2 above)
Comment (GerardoGomezG) [reply index.html]: I got it!
No further comments about this

**Review by GerardoGomezG — APPROVED** [review]
Hey @Albinpirela 
I've just checked the comments I left and you couldn't see. I saw the code was for the SVG images, so I don't need you to explain it anymore.
Keep working like this, you've got skills!

File: index.html (hunk 2 above)
Comment (Albinpirela) [reply index.html]: Hi, happy day! I hope you are very well, I'll explain, the difference is the color of the vector image from #ff8303 to #ffffff a small change at the last minute. 😄

File: index.html (hunk 1 above)
Comment (Albinpirela) [reply index.html]: Hello, exactly, they are xml code 💻 I await the approval. thank you

**Review by GerardoGomezG — APPROVED** [review]
Max-width 👍🏼 @Albinpirela

**Comment by Albinpirela** [comment]
## 
Captura de pantalla (57)
Captura de pantalla (58)
Captura de pantalla (59)
Add the work-about-me branch into develop

**Pull request merged**